### ISSUE TITLE
Lock Keys taking up space in gnome 3.34 w/ Wayland

### ISSUE THREAD

**Issue by deekej** [kazysmaster/gnome-shell-extension-lockkeys#77]
Hello! :)

I love your extension! Unfortunately after an update to Fedora 31, I got this small issue:
***The extension is taking up space, even though it is set to be hidden.***

Please, see the screenshot below for better explanation. When I hover above the empty space and click it, I can see this image:

![lock-keys-screenshot](https://user-images.githubusercontent.com/6081285/78769750-07b76d80-798e-11ea-915c-57e46e9929a3.png)

In previous Fedora versions, this space was not taken up by the extension (only when CapsLock / NumLock were turned on). On my small laptop screen this is taking up already scarce screen in the top bar. Could you please fix it? :)

**Comment by kazysmaster** [comment]
I see, I will look into this at some point.

**Comment by wangxa** [comment]
You are missing `this.panelButton.visible = numlock_state || capslock_state;` in `ShowhideIndicator`'s `ShowhideIndicator()` method.

**Comment by deekej** [comment]
I can confirm this patch works for me:

```diff
--- extension.js	2020-04-23 13:37:46.236361038 +0200
+++ extension.js.fixed	2020-04-23 13:37:34.609361645 +0200
@@ -285,6 +285,8 @@ ShowhideIndicator.prototype = {
 			this.capsIcon.show();
 		} else
 			this.capsIcon.hide();
+
+		this.panelButton.visible = numlock_state || capslock_state;
 	}
 }
```

Thanks @wangxa for your help! ;)
